Jenkinsfile.deploy

```diff
@@ -0,0 +1,75 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+pipeline {
+    agent {
+        label 'submarine-static || kie-rhel7'
```

Comment:
Can  we rename this to `kogito-static` instead?

Reply:
i guess we should, but this needs to be first done on label in Jenkins admin part